packages/db/src/schema/incident.ts

```diff
@@ -1,37 +1,53 @@
 import { relations, sql } from "drizzle-orm";
-import { integer, sqliteTable, text } from "drizzle-orm/sqlite-core";
+import {
+  integer,
+  primaryKey,
+  sqliteTable,
+  text,
+} from "drizzle-orm/sqlite-core";
 import { createInsertSchema, createSelectSchema } from "drizzle-zod";
+import * as z from "zod";
 
-import { page } from "./page";
+import { monitor } from "./monitor";
+import { workspace } from "./workspace";
 
+export const availableStatus = [
+  "investigating",
+  "identified",
+  "monitoring",
+  "resolved",
+] as const;
+
+export const StatusEnum = z.enum(availableStatus);
+
+// We should have a self relation. Such that we show the parent.
 export const incident = sqliteTable("incident", {
   id: integer("id").primaryKey(),
-  status: text("status", ["resolved", "investigating"]).notNull(),
+  status: text("status", ["resolved", "investigating"]).notNull(), // FIXME: delete from table!
+  title: text("title", { length: 256 }).notNull(),
 
-  pageId: text("page_id")
-    .notNull()
-    .references(() => page.id, { onDelete: "cascade" }),
+  workspaceId: integer("workspace_id").references(() => workspace.id),
 
-  createdAt: integer("updated_at", { mode: "timestamp" }).default(
+  createdAt: integer("created_at", { mode: "timestamp" }).default(
     sql`(strftime('%s', 'now'))`,
   ),
   updatedAt: integer("updated_at", { mode: "timestamp" }).default(
     sql`(strftime('%s', 'now'))`,
   ),
+  // createdBy
 });
 
 export const incidentUpdate = sqliteTable("incident_update", {
```

Comment:
We are not storing the `workspaceId` inside of it. Should we?

Reply:
Don't really understand why ?